### PR TITLE
Dataviews/Python README cleanup

### PR DESCRIPTION
**Pull request by dcbroad3**

### CODE REVIEW

File: basic_samples/DataViews/Python3/README.md [hunk 1]
```diff
@@ -29,8 +29,6 @@ or
 
 Automated test uses Python 3.6.8 x64
 
-Complete dependencies listed in [DEPENDENCIES.md](DEPENDENCIES.md)
```
Comment (derekendres) [inline basic_samples/DataViews/Python3/README.md:32]: we should probably include a LICENSE file that has the complete dependency list: using the blackduck readout https://osisoft.blackducksoftware.com/api/projects/2f22e568-1ba9-453f-864b-332bad4c4aa3/versions/5254b5b7-bf30-4937-9fbc-4d50a9880d53/reports

File: basic_samples/DataViews/Python3/README.md [hunk 2]
```diff
@@ -29,8 +29,6 @@ or
...
-
 For the general steps or switch languages see the Task [ReadMe](../)
```
Comment (derekendres) [inline basic_samples/DataViews/Python3/README.md:32]: I can't put this comment on the right line but on the line 5 we don't link to the library right

Comment (dcbroad3) [reply basic_samples/DataViews/Python3/README.md:32]: This should be fixed now

File: basic_samples/DataViews/Python3/README.md (hunk 1 above)
Comment (dcbroad3) [reply basic_samples/DataViews/Python3/README.md:32]: Moved LICENSE file to the right folder (had the wrong casing)

**Pull request merged**